pgmq-extension/sql/pgmq.sql

```diff
@@ -460,7 +460,7 @@ RETURNS TEXT AS $$
     extname = 'pg_partman';
 $$ LANGUAGE SQL;
 
-CREATE FUNCTION pgmq.drop_queue(queue_name TEXT, partitioned BOOLEAN DEFAULT FALSE)
+CREATE FUNCTION pgmq.drop_queue(queue_name TEXT, partitioned BOOLEAN)
```

Comment:
I think there may also need to be an `alter function`, or `drop function` and `create function` command added to `pgmq-extension/sql/pgmq--1.44--1.5.0.sq` since the signature is changing.